.chloggen/tcpcheckreceiver-add-tests.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: tcpcheckreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add test case for config.yaml
```

Comment:
a changelog entry goes to live in the changelog of the release, so we don't need one here as this is internal and won't matter to end users. You can remove this entry.